flipperdevices/flipperzero-firmware
# [FL-2249] Bluetooth: update status bar icon

# What's new

- Bluetooth: update status bar icon

# Verification 

- Compile and upload
- Use BLE to connect to flipper
- Check out our new icon

# Checklist (do not modify)
